README.md

```diff
@@ -137,7 +137,9 @@ if (socketjs.isSupported()) {
   var socket = socketjs.connect();
 
   // log messages as they arrive
-  socket.receive('greeting', console.log);
+  socket.receive('greeting', function (data) {
```

Comment:
For consistency with the rest of the codebase, can you remove the space after the `function` keyword?